frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTableConfig.tsx

```diff
@@ -99,178 +101,147 @@ const generateTableHeaders = (options: {
-                >
-                  ---
-                </span>
-                <ReactTooltip
-                  place="bottom"
-                  effect="solid"
-                  backgroundColor="#3e4771"
-                  id={`passing_${cellProps.row.original.id.toString()}`}
-                  data-html
-                >
-                  {getTooltip(cellProps.row.original.osquery_policy_ms)}
-                </ReactTooltip>
-              </>
-            );
-          },
-        },
-        {
-          title: "No",
-          Header: () => (
-            <>
-              <img alt="host passing" src={FailIcon} />
-              <span className="status-header-text">No</span>
-            </>
-          ),
-          disableSortBy: true,
-          accessor: "failing_host_count",
-          Cell: (cellProps: ICellProps): JSX.Element => {
-            if (cellProps.row.original.has_run) {
-              return (
-                <LinkCell
-                  value={`${cellProps.cell.value} host${
-                    cellProps.cell.value.toString() === "1" ? "" : "s"
-                  }`}
-                  path={
-                    PATHS.MANAGE_HOSTS +
-                    TAGGED_TEMPLATES.hostsByPolicyRoute(
-                      cellProps.row.original.id,
-                      PolicyResponse.FAILING,
-                      selectedTeamId
-                    )
-                  }
-                />
-              );
-            }
-            return (
-              <>
-                <span
-                  className="text-cell text-muted has-not-run tooltip"
-                  data-tip
-                  data-for={`failing_${cellProps.row.original.id.toString()}`}
-                >
-                  ---
-                </span>
-                <ReactTooltip
-                  place="bottom"
-                  effect="solid"
-                  backgroundColor="#3e4771"
-                  id={`failing_${cellProps.row.original.id.toString()}`}
-                  data-html
-                >
-                  {getTooltip(cellProps.row.original.osquery_policy_ms)}
-                </ReactTooltip>
-              </>
-            );
-          },
-        },
-        {
-          title: "Automations",
-          Header: "Automations",
-          disableSortBy: true,
-          accessor: "webhook",
-          Cell: (cellProps: ICellProps): JSX.Element => (
-            <StatusCell value={cellProps.cell.value} />
-          ),
-        },
-      ];
+          );
+        }
+        return (
+          <>
+            <span
+              className="text-cell text-muted has-not-run tooltip"
+              data-tip
+              data-for={`failing_${cellProps.row.original.id.toString()}`}
+            >
+              ---
+            </span>
+            <ReactTooltip
+              place="bottom"
+              effect="solid"
+              backgroundColor="#3e4771"
```

Comment:
Maybe replace with var colors since `colors` were introduced in this PR

Reply:
ah nice catch. will do.

Reply:
this is not in the new styleguide atm but I've brought asked @mike-j-thomas if we should change this value or update the styleguide? For now I'm going to leave as is.